lua/entities/sent_spawnpoint/init.lua

```diff
@@ -4,101 +4,103 @@ include( 'shared.lua' )
 
 -- Chat command config
 spawnPointCommands = {
-    ["unlinkSpawnPoint"] = { 
-        ["!unlinkspawn"] = true, 
-        ["!unlinkspawnpoint"] = true 
+    ["unlinkSpawnPoint"] = {
+        ["!unlinkspawn"] = true,
+        ["!unlinkspawnpoint"] = true
     },
-    ["unlinkThisSpawnPoint"] = { 
-        ["!unlinkthis"] = true 
+    ["unlinkThisSpawnPoint"] = {
+        ["!unlinkthis"] = true
     }
 }
 
 -- Helper Functions
 function createPlayerList( players )
     local playerList = {}
-    for _, player in pairs( players ) do
-        playerList[player] = true
+    for _, ply in pairs( players ) do
+        playerList[ply] = true
     end
 
     return playerList
 end
 
-function linkPlayerToSpawnPoint( player, spawnPoint )
-    player.LinkedSpawnPoint = spawnPoint
-    spawnPoint.LinkedPlayers[player] = "Linked"
+function linkPlayerToSpawnPoint( ply, spawnPoint )
+    ply.LinkedSpawnPoint = spawnPoint
+    spawnPoint.LinkedPlayers[ply] = "Linked"
 end
 
-function unlinkPlayerFromSpawnPoint( player, spawnPoint )
-    player.LinkedSpawnPoint = nil
-    spawnPoint.LinkedPlayers[player] = nil
+function unlinkPlayerFromSpawnPoint( ply, spawnPoint )
+    if not IsValid(ply) then return end
+
+    ply.LinkedSpawnPoint = nil
+    spawnPoint.LinkedPlayers[ply] = nil
```

Comment:
@PLally do you think this needs to run even if the player isn't valid?

Reply:
I don't think it will error if an invalid entity is passed,  maybe just check if its nil instead.